docs/csharp/whats-new/csharp-10.md

```diff
@@ -73,6 +75,16 @@ valid in C# 8.0 and later.
 
 For more information, see the [Extended property patterns](~/_csharplang/proposals/csharp-10.0/extended-property-patterns.md) feature proposal note. For more information about a property pattern, see the [Property pattern](../language-reference/operators/patterns.md#property-pattern) section of the [Patterns](../language-reference/operators/patterns.md) article.
 
+## Lambda expression improvements
+
+C# 10 includes many improvements to how lambda expressions are handled:
+
+- Lambda expressions may have a [natural type](../language-reference/operators/lambda-expressions.md#natural-type-for-lambda-expressions), where the compiler can infer a delegate type from the lambda expression or method group.
+- Lambda expressions may declare a [return type](../language-reference/operators/lambda-expressions.md#declared-return-type) when the compiler can't infer it.
+- [Attributes](../language-reference/operators/lambda-expressions.md#attributes) can be applied to lambda expressions.
+
+These features make lambda expressions more similar to methods and local functions. They make it easier to use lambda expressions without declaring a variable of a delegate type, and they work more seamlessly with the new ASP.NET minimal APIs.
```

Comment:
I'm not sure, but is "minimal" supposed to be title case? I have always referred to it as "ASP.NET Core Minimal APIs", and the word Core is missing.

```suggestion
These features make lambda expressions more similar to methods and local functions. They make it easier to use lambda expressions without declaring a variable of a delegate type, and they work more seamlessly with the new ASP.NET Core Minimal APIs.
```